share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
Though there is a minor issue. Since `subtitle` is set to have first letters of all words capitalized, URL that user searched for will not look exactly like the user had typed in the search box.

Here @moollaza 
![selection_137](https://cloud.githubusercontent.com/assets/5206277/16239481/a4fb3b16-3802-11e6-8332-62da54523871.png)